Formula/wuzz.rb

```diff
@@ -0,0 +1,80 @@
+require "language/go"
+
+class Wuzz < Formula
+  desc "Interactive cli tool for HTTP inspection"
```

Comment:
you might want to write either "`CLI`" or "`command-line`" for consistency.

Reply:
I just copy repo description of wuzz, I will change it to `CLI` for consistency.